dice-game/service/src/main.rs

```diff
@@ -0,0 +1,141 @@
+extern crate rand; // 0.6.5
+
```

Comment:
These 2 lines are not needed.

Reply:
addressed in https://github.com/oasislabs/tutorials/pull/55